deploy/eraser-config.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: v1
```

Comment:
did this file get generated?

Reply:
No, I created it myself so I can deploy the images to the clusters.